testing/tests/DevExpress.ui.widgets.form/form.tests.js

```diff
@@ -1407,6 +1411,28 @@ QUnit.test('optional mark aligned', function(assert) {
     assert.ok($optionalLabel.position().left < $optionalMark.position().left, 'optional mark should be after of the text');
 });
 
+[true, false].forEach(alignItemLabels => {
+    QUnit.test(`HtmlEditor with Toolbar is rendered inside form. alignItemLabels = ${alignItemLabels} (T986577)`, function(assert) {
+        const $form = $('#form').dxForm({
+            alignItemLabels,
+            width: 250,
+            items: [ {
+                label: { text: 'text' },
+                editorType: 'dxHtmlEditor',
+                editorOptions: {
+                    toolbar: { multiline: false, items: [ 'bold', 'italic', 'strike', 'underline' ] }
+                }
+            }, {
+                label: { text: 'Very very long text' }
+            } ]
+        });
+
+        const toolbar = $form.find(`.${TOOLBAR_CLASS}`).eq(0).dxToolbar('instance');
+        const menuItems = toolbar._getMenuItems();
+
+        assert.equal(menuItems.length, alignItemLabels ? 3 : 0, 'italic, strike and underline items are located in menu if alignItemLabels is enabled');
```

Comment:
>alignItemLabels ? 3 : 0

It's not clear why it should be '3'? Can HtmlEditor devs remove/redesign these items without fails in this test?